src/js/slider/slider.js

```diff
@@ -244,7 +244,7 @@ class Slider extends Component {
     const progress = this.getProgress();
 
     if (progress === this.progress_) {
-      return;
+      return progress;
```

Comment:
This function has a return value of `progress` so early returns should return that.